.github/workflows/security-scan.yml

```diff
@@ -11,10 +11,16 @@ env:
   ZAP_VERSION: 2.11.1
 
 jobs:
-  staging-dynamic-audit:
+  dynamic-audit:
     runs-on: ubuntu-latest
     env:
       DOCKER_APPLICATION_TAG: staging
```

Comment:
This isn't actually being used; it's being set [in the makefile](https://github.com/digital-land/performance-prototype/blob/41c2eb70a60ff1d147bcdb097bd74ccaf43552b4/Makefile#L102) which could be confusing if someone tries to change this

Reply:
Will remove from the makefile, currently the `docker-compose.security.yml` has it's own default so this should be set based on user input if needed.